modules/dhp_connected_devices/app/controllers/dhp_connected_devices/fitbit/fitbit_controller.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'fitbit/client'
+
+module DhpConnectedDevices
+  module Fitbit
+    class FitbitController < ApplicationController
+      include SentryLogging
+      before_action :feature_enabled
+
+      def connect
+        auth_url = fitbit_api.auth_url_with_pkce
+        redirect_to auth_url
+      end
+
+      def callback
+        auth_code = fitbit_api.get_auth_code(callback_params)
+        token_json = fitbit_api.get_token(auth_code)
+        storage_status = token_storage_service.store_tokens(@current_user, 'fitbit', token_json)
+        VeteranDeviceRecordsService.create_or_activate(@current_user, 'fitbit') unless storage_status == false
+        storage_status == true ? redirect_with_status('success') : redirect_with_status('error')
```

Comment:
If storage `status == false`, we redirect with status error which is great. But we don't actually know that this happened because we never log anything for this one case. Trying to think of a clean way to also log something to the extent of `"Token storage failed for user with ICN: #{@current_user&.icn}"` so that at least we know someone tried to connect their Fitbit and it failed on the token persistence step

Reply:
I'm open to it. Splitting this off as a tiny PR for the story.

Reply:
Narrator voice: we did not split this off. Raising and error instead of returning false

Reply:
Great, looks good. In this case, we can replace this line with 
```suggestion
redirect_with_status('success')
```
and we have one let scenario to test 😄